src/utils/file-upload.ts

```diff
@@ -109,3 +109,42 @@ export const uploadToUrl = async (url: string, file: File, key: string, userId:
     return response.status.value
   })
 }
+
+/**
+ * Uploads the specified file to Document Record Service.
+ * @param file the file to upload
+ * @param documentClass the document class defined for the document service. e.g. 'CORP'
+ * @param documentType the type of document. e.g. 'CNTA'
+ * @param businessId the business identifier(tempId or businessId)
+ * @returns a promise to return the axios response or the error response
+ */
+export const uploadDocumentToDRS = async (
+  document: File,
+  documentClass: string,
+  documentType: string,
+  businessId: string
+): Promise<ApiResponseIF> => {
+  const consumerFilingDate = new Date().toISOString()
+  const config = useRuntimeConfig()
+  const baseURL = config.public.docApiURL
+  const docApiKey = config.public.docApiKey
+  const docApiAccountId = config.public.docApiAccountId
```

Comment:
As mentioned above, i believe this is already in state and can be pulled from `useBcrosAccount().currentAccount.id`.